cmd/kured/main.go

```diff
@@ -300,12 +307,25 @@ func rebootAsRequired(nodeID string, window *timewindow.TimeWindow, TTL time.Dur
 		log.Fatal(err)
 	}
 
+	drainer := &kubectldrain.Helper{
+		Client:              client,
+		Force:               true,
+		DeleteLocalData:     true,
+		IgnoreAllDaemonSets: true,
+		ErrOut:              os.Stderr,
+		Out:                 os.Stdout,
+	}
+
```

Comment:
I think it would make the code clearer (if slightly longer) to create this as required in `drain()` and `uncordon()`

Reply:
Sure. It's just a different object to pass, not a big deal :)